src/Administration/Resources/app/administration/src/module/sw-first-run-wizard/view/sw-first-run-wizard-data-import/sw-first-run-wizard-data-import.html.twig

```diff
@@ -1,19 +1,33 @@
 {% block sw_first_run_wizard_data_import %}
 <div class="sw-first-run-wizard-data-import">
+    <div
+        v-if="installationError"
+        class="sw-first-run-wizard-plugin-installation__error"
+    >
+        <mt-banner
+            :title="pluginError.title"
+            variant="critical"
+        >
+            {{ pluginError.detail }}
+        </mt-banner>
+    </div>
+
     <sw-container
         v-if="!isInstallingPlugin"
         columns="1fr 1fr"
-        gap="24px 0"
+        gap="24px 14px"
```

Comment:
Just curious: is it possible to use the design tokens here? I guess in the end it's just a string passed as CSS right? Previously it was not possible to use SCSS variables but it might work with normal CSS variables 🤔